docs/data/release_archive.yml

```diff
@@ -428,6 +428,11 @@ release_archive:
       version: 1.0.0
       release_date: 2022-12-21
       filename: "opensearch"
+    - name: "Flink JDBC Connector"
+      connector: "jdbc"
+      version: 3.0.0
+      release_date: 2023-04-13
```

Comment:
Do we need to make sure that it's the same with the time we publish to Nexus?

Reply:
Add a FIXME note is more suggested